### PR TITLE
🔴 Urgent - Underline Buttons + Icons [Small - ish]

### PR DESCRIPTION
**Pull request by scalipsum**
**- Underline Button Style
- Icons to the buttons**

Screenshot 2023-06-13 at 18 04 37

### CODE REVIEW

File: Sources/Components/ARCButton.swift [hunk 1]
```diff
@@ -41,20 +45,30 @@ public struct ARCButton: View {
...
 
     public var body: some View {
         Button(action: onTap) {
             ZStack {
-                Text(title)
-                    .style(.arcButton)
-                    .foregroundColor(buttonStyle.textColor)
-                    .opacity(isLoading ? 0 : 1)
+                HStack(spacing: 0) {
+                    if let icon = icon {
```
Comment (BenShutt) [inline Sources/Components/ARCButton.swift]: Teeny, can simplify slightly with `if let icon`

File: Sources/Components/ARCButton.swift [hunk 2]
```diff
@@ -41,20 +45,30 @@ public struct ARCButton: View {
...
+                        icon
+                            .frame(maxWidth: .ArcButton.iconSize, maxHeight: .ArcButton.iconSize)
```
Comment (BenShutt) [inline Sources/Components/ARCButton.swift]: Do we want just
```
.frame(size: yourSize)
```

File: Sources/Components/ARCButton.swift [hunk 3]
```diff
@@ -41,20 +45,30 @@ public struct ARCButton: View {
...
+                            .scaledToFit()
```
Comment (BenShutt) [inline Sources/Components/ARCButton.swift:64]: Q rather than a suggestion, should we use `.resizeable()` before this?

Comment (scalipsum) [reply Sources/Components/ARCButton.swift:64]: Yup, required, nice spot. It enables it to scale nicely.

**Review by BenShutt — APPROVED** [review]

**Pull request merged**